benchmark/solver/solver.cpp

```diff
@@ -70,12 +70,13 @@ DEFINE_bool(
     rel_residual, false,
     "Use relative residual instead of residual reduction stopping criterion");
 
-DEFINE_string(solvers, "cg",
-              "A comma-separated list of solvers to run. "
-              "Supported values are: bicgstab, bicg, cb_gmres_keep, "
-              "cb_gmres_reduce1, cb_gmres_reduce2, cb_gmres_integer, "
-              "cb_gmres_ireduce1, cb_gmres_ireduce2, cg, cgs, fcg, gmres, idr, "
-              "lower_trs, upper_trs, overhead");
+DEFINE_string(
+    solvers, "cg",
+    "A comma-separated list of solvers to run. "
+    "Supported values are: bicgstab, bicg, cb_gmres_keep, "
+    "cb_gmres_reduce1, cb_gmres_reduce2, cb_gmres_integer, "
+    "cb_gmres_ireduce1, cb_gmres_ireduce2, cg, cgs, fcg, minres, gmres, idr, "
```

Comment:
I think we try to use alphabetical order here, or?

Reply:
Right, I guess I wasn't paying attention.